lib/src/mixins/batched_redraws.dart

```diff
@@ -6,23 +6,42 @@ import 'dart:html';
 import 'package:react/react.dart' as react;
 
 class _RedrawScheduler implements Function {
-  Set<react.Component> _components = new Set();
+  Map<react.Component, List<Function>> _components =
+      <react.Component, List<Function>>{};
```

Comment:
This is super confusing, does this just mean `new Map<react.Component, List<Function>>()`? That might read a little more clearly.